cmd/proxy-endpoint/callback_server.go

```diff
@@ -0,0 +1,123 @@
+package main
```

Comment:
I think we can move this application to a separate repo to prevent making monorepo

Reply:
I think it could be useful only for integration testing, and since all other will depend on this repo I think we could reuse this apps from here?